internal/command/get.go

```diff
@@ -42,8 +44,9 @@ func (c *GetCommand) Run(args []string) int {
 	}
 
 	path = c.normalizePath(path)
-
-	abort, diags := getModules(ctx, &c.Meta, path, testsDirectory, update)
+	// mdTODO: do I need to come back here and handle the deprecations? Could be useful even if not directly in scope.
+	abort, diags, deprecations := getModules(ctx, &c.Meta, path, testsDirectory, update)
+	log.Printf("[INFO] : %s", deprecations)
```

Comment:
This actually pertains to an open question I have 😄. Was wondering whether it'd make sense to also include the module deprecation warning for `get` as well as `init`. Was figuring we might as well since they'd be already present.

Currently have the consolidated warning being generated and appended to the other diags in `*ModuleInstaller.InstallModules`– so we'd get the deprecations warnings implicitly in `get`. Any issue with this?